cmd/k8s-bigip-ctlr/main.go

```diff
@@ -108,6 +108,7 @@ var (
 	as3Validation      *bool
 	sslInsecure        *bool
 	trustedCertsCfgmap *string
+	as3Mode            *bool
```

Comment:
Instead of a boolean, let's use a string input called "agent", which by default set to "cccl". Users can override this to "as3" and later we can set this as a default value. It can accommodate other implementations too.

Reply:
@mdditt2000 Thoughts?

Reply:
Done the changes as suggested using a string parameter agent